packages/nitro/src/rollup/plugins/externals.ts

```diff
@@ -67,6 +69,12 @@ export function externals (opts: NodeExternalsOptions): Plugin {
     },
     async buildEnd () {
       if (opts.trace !== false) {
+        for (const pkgName of opts.traceInclude || []) {
+          const path = await this.resolve(pkgName)
+          if (path.id) {
```

Comment:
Resolve result can be possibly `null`
```suggestion
          if (path?.id) {
```